From 7a75a8a0f7089b716e702e6bb0cf13fc3194e3db Mon Sep 17 00:00:00 2001
From: Nick Schermer <nick@xfce.org>
Date: Sun, 29 Apr 2012 13:35:37 +0200
Subject: [PATCH] Remove gnome-keyring remains.

Not required anymore. gnome_keyring_daemon_prepare_environment_sync
is a stub function that does nothing.

Upstream-Status: backport [1]

[1] http://git.xfce.org/xfce/xfce4-session/commit/?id=7a75a8a0f7089b716e702e6bb0cf13fc3194e3db
---
 configure.in                   |    5 -----
 xfce4-session/Makefile.am         |    2 --
 xfce4-session/xfsm-compat-gnome.c |    8 --------
 3 files changed, 0 insertions(+), 15 deletions(-)

diff --git a/configure.in b/configure.in
index f8684b3..9aa4f8f 100644
--- a/configure.in
+++ b/configure.in
@@ -96,11 +96,6 @@ XDT_CHECK_PACKAGE([DBUS], [dbus-1], [1.1.0])
 XDT_CHECK_PACKAGE([DBUS_GLIB], [dbus-glib-1], [0.84])
 XDT_CHECK_PACKAGE([XFCONF], [libxfconf-0], [4.9.0])
 
-dnl Check for gnome support
-XDT_CHECK_OPTIONAL_PACKAGE([GNOME_KEYRING], [gnome-keyring-1], [2.22],
-                           [libgnome-keyring],
-                           [Extra gnome-keyring compatibility support], [no])
-
 dnl Check for debugging support
 XDT_FEATURE_DEBUG([xfsm_debug_default])
 
diff --git a/xfce4-session/Makefile.am b/xfce4-session/Makefile.am
index 5e8343a..c015154 100644
--- a/xfce4-session/Makefile.am
+++ b/xfce4-session/Makefile.am
@@ -66,7 +66,6 @@ xfce4_session_SOURCES =							\
 	xfsm-upower.h
 
 xfce4_session_CFLAGS =							\
-	$(GNOME_KEYRING_CFLAGS)						\
 	$(LIBSM_CFLAGS)							\
 	$(LIBX11_CFLAGS)						\
 	$(LIBXFCE4UI_CFLAGS)						\
@@ -93,7 +92,6 @@ xfce4_session_LDADD =							\
 	$(DBUS_GLIB_LIBS)						\
 	$(LIBWNCK_LIBS)							\
 	$(XFCONF_LIBS)							\
-	$(GNOME_KEYRING_LIBS)						\
 	-lm
 
 xfce4_session_DEPENDENCIES =						\
diff --git a/xfce4-session/xfsm-compat-gnome.c b/xfce4-session/xfsm-compat-gnome.c
index b5b4a99..e1fa16f 100644
--- a/xfce4-session/xfsm-compat-gnome.c
+++ b/xfce4-session/xfsm-compat-gnome.c
@@ -59,10 +59,6 @@
 
 #include <libxfce4util/libxfce4util.h>
 
-#ifdef HAVE_GNOME_KEYRING
-#include <gnome-keyring.h>
-#endif
-
 #include <xfce4-session/xfsm-compat-gnome.h>
 
 #define GNOME_KEYRING_DAEMON "gnome-keyring-daemon"
@@ -154,10 +150,6 @@ gnome_keyring_daemon_startup (void)
             }
 
           g_strfreev (lines);
-
-#ifdef HAVE_GNOME_KEYRING
-          gnome_keyring_daemon_prepare_environment_sync ();
-#endif
         }
       else
         {
-- 
1.7.4.4

